src/portable/portable.c

```diff
@@ -44,6 +45,9 @@ static const char profile_dirs[] = CONF_PATHS_NULSTR("systemd/portable/profile")
 #define PORTABLE_DROPIN_MARKER_BEGIN "# Drop-in created for image '"
 #define PORTABLE_DROPIN_MARKER_END "', do not edit."
 
+/* Maximum length of an extended attribute is 64KB */
+#define XATTR_MAX 64U * 1024U
```

Comment:
enclose in `()` to make safe to include in arbitrary expressions

Reply:
also, maximum length of an selinux label is NAME_MAX, see server_process_datagram() comment